src/routes/docs/products/databases/+layout.svelte

```diff
@@ -102,9 +108,39 @@
             ]
         }
     ];
+
+    const legacyUrl = $derived(
+        page.url.pathname
+            .replace('/products/databases', '/products/databases/legacy')
+            .replace('rows', 'documents')
+            .replace('tables', 'collections')
+    );
+
+    const hideSubtitleRoutes = ['offline', 'backups', 'csv-imports'];
```

Comment:
@adityaoberai we excluded these pages. context: the copying of related files into legacy ones.